contrib/openbsm/bin/bsmconv/linau.h

```diff
@@ -0,0 +1,120 @@
+#ifndef _LINAU_H_
+#define _LINAU_H_
+
+#include <sys/nv.h>
+#include <sys/queue.h>
+
+#include <stdint.h>
+#include <stdio.h>
+
+
+struct linau_event {
+	TAILQ_HEAD(, linau_record) 	le_records;
+};
+
+struct linau_record {
+	char				*lr_type;
+	uint32_t			 lr_id;
```

Comment:
Done.

The previous order originated from the fact that `type` comes before `id` in a Linux Audit record.